packages/nextjs/package.json

```diff
@@ -29,6 +28,7 @@
     "@sentry/types": "6.3.1",
     "@types/webpack": "^5.28.0",
     "eslint": "7.20.0",
+    "next": "^10.1.3",
```

Comment:
Why is `next` a dev dependency? Can this cause any problems in projects where the `next` version doesn't match `^10.1.3`?

Reply:
We need it for the request/response types in the `withSentry` handler.

Reply:
dev dependencies will not be installed on a consumer side, so it should not bring any problem. If `next` is needed in the development of the repo then dev deps is the right place.